Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+  && apt-get install -y python3-pip python3-dev \
```

Comment:
does this need to be specific about which python3 version to use? I think some of the dependencies here require > 3.6.

```dockerfile
RUN apt-get update && apt-get install -y python3.9 python3.9-dev python3-pip
```

Reply:
I switched to python:3.10-slim-bullseye. So we will be using 3.10.